solr/src/main/java/dk/dbc/opensearch/solr/profile/Profiles.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (C) 2018 DBC A/S (http://dbc.dk/)
+ *
+ * This is part of opensearch-solr
+ *
+ * opensearch-solr is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * opensearch-solr is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package dk.dbc.opensearch.solr.profile;
+
+import dk.dbc.opensearch.cql.CQLException;
+import dk.dbc.opensearch.input.badgerfish.BadgerFishReader;
+import dk.dbc.opensearch.solr.config.FieldSpec;
+import dk.dbc.opensearch.solr.flatquery.FlatQueryOr;
+import dk.dbc.opensearch.solr.flatquery.FlatQuerySearch;
+import dk.dbc.opensearch.solr.SolrRules;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ *
+ * @author DBC {@literal <dbc.dk>}
+ */
+public class Profiles implements Serializable {
```

Comment:
_(Comment suggestion)_
A collection of OpenAgency profiles to be used for a request.

A number og OpenAgency profiles, can be given in the request, and must be held for further processing.